R/examples.R

```diff
@@ -1,16 +1,37 @@
 #' Example ID
 #' @description
-#' This function returns examples of working IDs that can be used with the API.
+#' This function returns examples of working IDs that can be used with the eesyapi functions, such
+#' as query_dataset().
 #'
-#' @param level Level of ID example to return: "publication" or "data set"
-#' @param environment Environment to return a working example for
-#' @param group Choose the publication group of examples to use. Can be "attendance".
+#' @param level Level of ID example to return. A range are available, although not every example
+#' group necessarily contains all possible examples. The following are generally available.
+#'   - "all": Return full list of example options for given group.
+#'   - "publication": Return example publication ID
+#'   - "dataset": Return example data set ID
+#'   - "time_period" / "time_periods": Return example time_period(s)
+#'   - "location_id" / "location_ids": Return example location ID(s)
+#'   - "location_code": Return example location code
```

Comment:
Should it be `location code(s)` to match the pattern from other lines?

Reply:
That would make sense if I'd got round to adding multiple location codes. I've just added some so that it does make sense and updated this line accordingly.